src/bci_build/package/firefox.py

```diff
@@ -9,6 +9,7 @@
 from bci_build.package import ParseVersion
 from bci_build.package import Replacement
 from bci_build.package.kiosk import KIOSK_EXCLUSIVE_ARCH
+from bci_build.package.kiosk import KioskRegistry
 
 FIREFOX_CONTAINERS = [
     ApplicationStackContainer(
```

Comment:
I'd suggest to strip the `kiosk-` part from the name (at least for non-TW), as the registry url now ends with `kiosk/kiosk-firefox-esr`